### PR TITLE
stripe-cli: 1.21.10 -> 1.22.0

### PR DESCRIPTION
**Pull request by afh**
Release notes: https://github.com/stripe/stripe-cli/releases

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by afh** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 361049`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>stripe-cli</li>
  </ul>
</details>

**Comment by afh** [comment]
@ofborg: eval

**Comment by afh** [comment]
@ofborg: eval

File: pkgs/by-name/st/stripe-cli/package.nix [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
+  version = "1.22.0";
 
   src = fetchFromGitHub {
     owner = "stripe";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-L2ydkrm2foSIh67LsMYNvV53Hdj9TdV7C4tKeHe+AK0=";
+    hash = "sha256-h5Q9+WoMJ5swmyL4GIiWZ9HDnr/w23iBVtp4EQ1yYn0=";
   };
   vendorHash = "sha256-TuxYJ3u4/5PJYRoRgom+M1au9XerZ+vj9X3jUWTPM58=";
```
Comment (06kellyjac) [inline pkgs/by-name/st/stripe-cli/package.nix:13]: I'm amazed the dependencies haven't been updated for 6 months...

**Review by 06kellyjac — APPROVED** [review]
LGTM

**Review by kashw2 — APPROVED** [review]

File: pkgs/by-name/st/stripe-cli/package.nix (hunk 1 above)
Comment (afh) [reply pkgs/by-name/st/stripe-cli/package.nix:13]: As am I 🙂

I took the liberty of adding a minor changes that removes the use of `pname` for the `repo` attribute.

**Review by 06kellyjac — APPROVED** [review]

**Comment by thiagokokada** [comment]
@ofborg eval

**Comment by afh** [comment]
Thanks for merging this, @RaghavSood, this is very helpful 👍

**Pull request merged**